### DISCUSSION TITLE
支持执行Unity的.anim格式动画么？或是支持创建AnimationClip？有案例么？

### DISCUSSION THREAD

**Discussion by zoyoy1203** [galacean/discussion#57] [Engine] [answered]
支持执行Unity的.anim格式动画么？或是支持创建AnimationClip？有案例么？

**Comment by GuoLei1990** [comment]
We have no plans to support any of Unity's assets at all.
For how to produce animation files, one is that glTF will be automatically parsed into AniamtionClip, and the other is created through the editor, but it needs to wait for the editor to be published.

**Comment by GuoLei1990** [comment, marked as answer]
0.9 里程碑后支持自定义 AnimationClip，案例： https://oasisengine.cn/#/examples/latest/animation-customAnimationClip
